examples/kms/vault/vault.yaml

```diff
@@ -60,60 +60,54 @@ spec:
               name: vault-api
 ---
 apiVersion: v1
-items:
-  - apiVersion: v1
-    data:
-      init-vault.sh: |
-        set -x -e
-
-        timeout 300 sh -c 'until vault status; do sleep 5; done'
+kind: ConfigMap
+metadata:
+  name: init-scripts
```

Comment:
Should we keep the creationTimestamp to be `null` as the previous yaml? Since it a configmap, it doesn't make much sense to keep track of its creation time. Wdyt?

Reply:
On the other hand,  it's does not matter at all. Looks good to me ;)

Reply:
The `creationTimestamp` gets set automatically if missing, I think. In any case, it does not add any value for the example or tests.